researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -47,27 +47,23 @@
 public abstract class BridgeDataProvider extends DataProvider {
     private static final Logger logger = LoggerFactory.getLogger(BridgeDataProvider.class);
 
-    protected final Gson gson = RestUtils.GSON;
-
     // set in initialize
-    protected UserLocalStorage userLocalStorage;
-    protected ConsentLocalStorage consentLocalStorage;
     protected TaskHelper taskHelper;
     protected UploadHandler uploadHandler;
 
+    private final StorageAccessWrapper storageAccessWrapper;
```

Comment:
What's the difference between ResearchStackDAO which contains User, and the AccountDAO?  They seem like they are duplicated in some ways.